mkdocs.yml

```diff
@@ -1162,6 +1162,7 @@ nav:
         - integrations/builtin/credentials/serp.md
         - integrations/builtin/credentials/servicenow.md
         - integrations/builtin/credentials/sms77.md
+        - integrations/builtin/credentials/solarwindsipam.md
```

Comment:
Same here as the other PR in terms of ordering. The `sms77` is not directly in front of this one alphabetically when you take the display value into account.